config.py

```diff
@@ -87,11 +87,19 @@ class Config(object):
     def init_app(app):
         repo_root = os.path.abspath(os.path.dirname(__file__))
         template_folders = [
-            os.path.join(repo_root, 'app/templates')
+            os.path.join(repo_root, 'app/templates'),
+            os.path.join(repo_root, 'node_modules/govuk-frontend/'),
+            os.path.join(repo_root, 'node_modules/govuk-frontend/components')
```

Comment:
Are we planning to go down the route of linking directly to `node_modules` files here, rather than copying them across to the `app/` folder in a gulpfile?

Reply:
Could do either way.

This PR is more for my reference than anything, I need to improve how we integrate the new Jinja extension with Flask.

Reply:
I’m not sure a see the value in copying with gulp?